### PR TITLE
[dotnet-msidentity] help, README, and version update

### PR DESCRIPTION
**Pull request by deepchoudhery**
Update
- both READMEs in src/tools/dotnet-msidentity and src/MsIdentity (package version in the README as well)
- help text in src/tools/dotnet-msidentity/Program.cs

### CODE REVIEW

File: tools/dotnet-msidentity/dotnet-msidentity.csproj [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <Product>Microsoft identity platform app registration tool</Product>
     <Description>
       This package is a dotnet global tool which registers new Azure AD or Azure AD B2C applications, and updates your code accordingly, or modifies/edits existing
```
Comment (sayedihashimi) [inline tools/dotnet-msidentity/dotnet-msidentity.csproj:28]: We should mention somewhere in here that this tool only work with asp.net core projects.

File: tools/dotnet-msidentity/README.md [hunk 2]
```diff
@@ -199,6 +219,6 @@ The tool supports ASP.NET Core applications created with .NET 5.0 and netcoreapp
 
 ```Shell
 dotnet new blazorwasm --auth SingleOrg --framework netcoreapp3.1
-dotnet-msidentity
+dotnet msidentity
 dotnet run -f netstandard2.1
```
Comment (sayedihashimi) [inline tools/dotnet-msidentity/README.md:223]: It looks like on line `221` we tell them to create the project with `--framework netcoreapp3.1` and the on line `222` we tell them to run with 2.1 `dotnet run -f netstandard2.1`. That doesn't seem right.

File: src/MSIdentityScaffolding/README.md [hunk 3]
```diff
@@ -1,7 +1,11 @@
 # dotnet-msidentity
 Command line tool that creates Microsoft identity platform applications in a tenant (AAD or B2C) and updates the configuration code of your ASP.NET Core applications (mvc, webapp, blazorwasm, blazorwasm hosted, blazorserver). The tool can also be used to update code from an existing AAD/AAD B2C application.
 
-## Installing/Uninstalling the tool
+## Installing/Uninstalling the release/prerelease tool
```
Comment (sayedihashimi) [inline src/MSIdentityScaffolding/README.md:4]: I think the previous title was better. Adding `release/prerelease` isn't adding value here. Later if we add instructions for both release & prerelease we will just do sub-headings with 3 `#` symbols.

File: src/MSIdentityScaffolding/README.md [hunk 4]
```diff
@@ -21,7 +25,7 @@ Have an AAD or B2C tenant (or both).
 
 ```text
 dotnet-msidentity:
```
Comment (sayedihashimi) [inline src/MSIdentityScaffolding/README.md:27]: Should this be `dotnet msidentity`?

File: src/MSIdentityScaffolding/README.md [hunk 5]
```diff
@@ -34,44 +38,64 @@ Usage:
   dotnet-msidentity [command] [options]
```
Comment (sayedihashimi) [inline src/MSIdentityScaffolding/README.md:38]: Should this be `dotnet msidentity`?

File: src/MSIdentityScaffolding/README.md [hunk 6]
```diff
@@ -34,44 +38,64 @@ Usage:
...
+  
   --list--aad-apps                     Lists AAD Applications for a given tenant + username.
+  
   --list-service-principals            Lists AAD Service Principals for a given tenant + username.
+  
   --list-tenants                       Lists AAD + AAD B2C tenants for a given username.
+  
   --update-project                     Given client id for an Azure AD/AD B2C app, update appsettings.json, local user secrets. [TODO : and project code(Startup.cs, project references to get the app auth ready).]
+  
+  -- create-client-secret              Create a Client secret for given app registration (client id) and print the secret.
```
[the remark this answers is not in the document]
Comment (deepchoudhery) [reply src/MSIdentityScaffolding/README.md:60]: sounds good

File: src/MSIdentityScaffolding/README.md (hunk 5 above)
Comment (deepchoudhery) [reply src/MSIdentityScaffolding/README.md:38]: both  "dotnet msidentity" and "dotnet-msidentity" work but "dotnet msidentity" is more consistent with our other tools.

File: tools/dotnet-msidentity/dotnet-msidentity.csproj (hunk 1 above)
Comment (deepchoudhery) [reply tools/dotnet-msidentity/dotnet-msidentity.csproj:28]: added "Works with ASP.NET Core 3.1 and .NET 5.0, 6.0 projects."

Comment (sayedihashimi) [reply tools/dotnet-msidentity/dotnet-msidentity.csproj:28]: Ok that is good, I may change it later when I do the overall review of the content.

**Pull request merged**